strr-examiner-web/app/components/Host/Expansion/FilingHistory.vue

```diff
@@ -29,6 +29,8 @@ const { data: filingHistory, status } = await useLazyAsyncData<FilingHistoryEven
         getRegistrationFilingHistory((activeRecord.value as HousRegistrationResponse).id)
       ])
       allFilingHistory = [...applicationHistory, ...registrationHistory]
+      // sort both histories by date
+      allFilingHistory.sort((a, b) => new Date(a.createdDate).getTime() - new Date(b.createdDate).getTime())
```

Comment:
Shouldn't this be outside the if else block? 
Just before this, https://github.com/bcgov/STRR/blob/8b5d7c3688f6e5a077344a73f3bcca31a0ce81b9/strr-examiner-web/app/components/Host/Expansion/FilingHistory.vue#L37

Reply:
Good catch, thank you, updated.